src/test/java/com/stock/stocklist/service/StockListServiceTest.java

```diff
@@ -46,4 +49,42 @@ class StockListServiceTest {
         verify(stockListMapper, times(1)).findById(99);
     }
 
+    @Test
+    public void クエリパラメータを指定しなかったときにfindAllメソッドが呼び出されること() throws Exception {
+        List<StockList> stockList = List.of(
+                new StockList(1, "メタノール", "HPLC用", 3, "L", LocalDate.parse("2023-05-24")),
+                new StockList(2, "塩化カリウム", "特級", 500, "g", LocalDate.parse("2023-07-19")),
+                new StockList(3, "硫酸ナトリウム", "特級", 5, "kg", LocalDate.parse("2023-10-11"))
+        );
+
+        doReturn(stockList).when(stockListMapper).findAll();
+
+        List<StockList> actual = stockListService.findData(null);
+        assertThat(actual).isEqualTo(stockList);
+        verify(stockListMapper, times(1)).findAll();
+        verify(stockListMapper, never()).findByName("メタノール");
+
+    }
+
+    @Test
+    public void 存在する名前をクエリパラメータに指定したときにfindByNameメソッドが呼び出されること() throws Exception {
```

Comment:
クエリパラメータはリクエストを受け付けるControllerでの用語なので業務ロジックを担当するServiceではあまり意識する必要のない言葉ですね。
また、戻り値についてもテストケースで言及するとよいです。
```suggestion
    public void 存在する名前を指定したときにfindByNameメソッドが呼び出されて在庫情報が返却されること() throws Exception {
```
というようにするのが良いかと思います。

Reply:
e79b24306062d051bb7f653d70f7028fbfd424bc
対応いたしました。
findAll()メソッドについても変更しています。
ご確認お願いいたします。